conf/perf_tests.yaml.template

```diff
@@ -57,6 +57,11 @@ ui:
                 customization_templates: 10
                 system_image_types: 10
                 iso_datastores: 10
+        physical_infra:
```

Comment:
**REQUIRED:** You don't need to add this, we won't be doing anything with perf tests for a while and this is actually perhaps old system.

Reply:
OK, Thanks @psav. I will remove this.